docs/source/contributing/writing-docs.adoc

```diff
@@ -159,3 +160,11 @@ $ rake build
 ----
 
 If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+[[building-openshift-docs-automatically]]
+=== Building the OpenShift Documentation Automatically
+
+To automatically get the view of {project} documentation integrated into the OpenShift documentation, open the link:https://ci.centos.org/job/minishift-docs/[minishift-docs] job in your browser and trigger it by clicking on `Build with Parameters` and specifying your repo URL in `REPO` and branch name containing your documentation changes in `BRANCH` parameters respectively.
+This job will perform all necessary steps including the steps mentioned in xref:manually-building-openshift-docs[Manually Building the OpenShift Documentation].
+
+At the end of the successful job, you will receive a message with the location of the hosted {project} documentation.
```

Comment:
Receive a message how? I guess you mean the build log will contain the URL? Can we not display a generic form of the URL as we do in other places?